### ISSUE TITLE
IONIC 4 "no-margin" CSS attribute is deprecated

### ISSUE THREAD

**Issue by cbleu** [eakoriakin/ionic-selectable#199]
Just so you know I am getting this console warning on pages that I'm using the select-searchable component. Not sure if his may become an issue in the future.

```console
[DEPRECATED][CSS] Ionic CSS attributes are deprecated.
Replace:
'<ion-list no-margin>'

With:
'<ion-list class="ion-no-margin">'
```

**Comment by edy-ap** [comment]
I've added a pr to fix it
#206

**Comment by edy-ap** [comment]
The breaking changes were already fixed in version [4.5.0](https://github.com/eakoriakin/ionic-selectable/releases/tag/4.5.0)
